champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/ranger/sword/Bullseye.java

```diff
@@ -138,11 +145,15 @@ public void updateCharge() {
                 continue;
             }
             // Spawn particles
+            if (player == null) return;
```

Comment:
Unnecessary check, its already being done here

`Bukkit.getPlayer(playerUUID) == null`

There is a little double up there but not too concerned about it